src/ppl/src/SimulatedAnnealing.h

```diff
@@ -51,6 +51,16 @@ class Logger;
 }  // namespace utl
 
 namespace ppl {
+
+struct DebugSettings
+{
+  std::unique_ptr<AbstractIOPlacerRenderer> renderer_;
+  int painting_interval_ = 0;
+  bool isNoPauseMode_ = false;
```

Comment:
Use the same name pattern as the other structs. They use the underline as a separator and don't have the underline at the end.
`painting_interval_` --> `painting_interval`
`isNoPauseMode_` --> `is_no_pause_mode`

Reply:
I followed the pattern used in FastRoute:
https://github.com/The-OpenROAD-Project/OpenROAD/blob/2313c4895460a815bf1bce666ded4a52e34074f2/src/grt/src/fastroute/include/FastRoute.h#L78C1-L89C2

Should I change it either way?

Reply:
This usage in FastRoute needs to be corrected. I will update it when I have time. Follow the pattern we have in IOPlacer for structs:
https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ppl/src/Slots.h#L70-L84